python/ray/tests/test_get_locations.py

```diff
@@ -0,0 +1,78 @@
+import numpy as np
+import pytest
+import platform
+import time
+
+import ray
+
+
+def test_uninitialized():
+    with pytest.raises(RuntimeError):
+        ray.util.get_object_locations([])
+
+
+def test_get_locations(ray_start_regular):
+    node_id = ray.runtime_context.get_runtime_context().get()["node_id"]
+    sizes = [100, 1000]
+    obj_refs = [ray.put(np.zeros(s, dtype=np.uint8)) for s in sizes]
+    ray.wait(obj_refs)
+    locations = ray.util.get_object_locations(obj_refs)
+    for idx, location in enumerate(locations):
+        assert location.primary_node_id() == node_id
+        # assert location.object_size() == sizes[idx]
```

Comment:
this doesn't work too. It seems to me the object_size() is larger than the data we put in, my suspicion it it includes the metadata as part of data size too. Are we ok this this semantic?

Reply:
I think we should return data size + metadata size. Maybe test if the data size is within 5% range or something?